README.md

```diff
@@ -1 +1,27 @@
-# trial-project
\ No newline at end of file
+# Review one of my trial project
+
+> I am in role of the Code Reviewer
+
+![screenshot](./scrnsht_trial_project.png)
+
+👤 **Elamar Abdulkarimov**
+
+- GitHub: [@elmar8287](https://github.com/elmar8287)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
```

Comment:
A nice tip to consolidate your reviews under a single summary is to click the "Start a review" button when adding the first review comment. This helps to house all your inline comment under one single summary like this current review. :wink:
This is an example inline comment. :slightly_smiling_face:
You can see how it's under the main summary above and point to a specific part of the code where you want to suggest changes or just comment on it generally. :wink:
To do this, make sure you go to the "File changes" tab and use the "Start a review" button for your first review comment.
- [ ] The project should have a descriptive README.md ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)).
A good README should have a description that is clear, short, and to the point. It should describe the importance of your project, what it does and how to use it. This way technical and non-technical users can understand how to use it.  Refer to this template [here](https://github.com/microverseinc/readme-template) and customise it to your needs.